src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -39,7 +39,7 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
     constructor(parent: AzureParentTreeItem, server: Server) {
         super(parent);
         this.server = server;
-        this._keytar = tryGetKeyTar();
+        this._keytar = ext.keytar;
```

Comment:
Since keytar is on `ext` now, we don't need to store it in this class and we can just use `ext.keytar` directly